pkg/applicationserver/io/web/redis/registry.go

```diff
@@ -53,6 +53,10 @@ type WebhookRegistry struct {
 	Redis *ttnredis.Client
 }
 
+func (r *WebhookRegistry) allKey(ctx context.Context) string {
```

Comment:
This key/index does not exist right now, so you will either need a migration or to actually scan the database.

Checkout how `ns-db` does this here:
https://github.com/TheThingsNetwork/lorawan-stack/blob/865c1a18686d1c1d9d93240e5bf539db2b6fd7df/cmd/ttn-lw-stack/commands/ns_db.go#L49
https://github.com/TheThingsNetwork/lorawan-stack/blob/865c1a18686d1c1d9d93240e5bf539db2b6fd7df/cmd/ttn-lw-stack/commands/redis.go#L73